tools/trace2syz/trace2syz/util.go

```diff
@@ -0,0 +1,857 @@
+		"BPF_MSH":                                             0xa0,
+		"BPF_ADD":                                             0,
+		"BPF_SUB":                                             0x10,
+		"BPF_MUL":                                             0x20,
+		"BPF_DIV":                                             0x30,
+		"BPF_OR":                                              0x40,
+		"BPF_AND":                                             0x50,
+		"BPF_LSH":                                             0x60,
+		"BPF_RSH":                                             0x70,
+		"BPF_NEG":                                             0x80,
+		"BPF_MOD":                                             0x90,
+		"BPF_XOR":                                             0xa0,
+		"BPF_JA":                                              0x00,
+		"BPF_JEQ":                                             0x10,
+		"BPF_JGT":                                             0x20,
+		"BPF_JGE":                                             0x30,
+		"BPF_JSET":                                            0x40,
+		"BPF_ALU64":                                           0x07,
+		"BPF_DW":                                              0x18,
+		"BPF_XADD":                                            0xc0,
+		"BPF_MOV":                                             0xb0,
+		"BPF_ARSH":                                            0xc0,
+		"BPF_END":                                             0xd0,
+		"BPF_TO_LE":                                           0x00,
+		"BPF_TO_BE":                                           0x08,
+		"BPF_FROM_LE":                                         0x00,
+		"BPF_FROM_BE":                                         0x08,
+		"BPF_JNE":                                             0x50,
+		"BPF_JLT":                                             0xa0,
+		"BPF_JLE":                                             0xb0,
+		"BPF_JSGT":                                            0x60,
+		"BPF_JSGE":                                            0x70,
+		"BPF_JSLT":                                            0xc0,
+		"BPF_JSLE":                                            0xd0,
+		"BPF_CALL":                                            0x80,
+		"BPF_EXIT":                                            0x90,
+
+		"BPF_REG_0":           0x0,
+		"BPF_REG_1":           0x1,
+		"BPF_REG_2":           0x2,
+		"BPF_REG_3":           0x3,
+		"BPF_REG_4":           0x4,
+		"BPF_REG_5":           0x5,
+		"BPF_REG_6":           0x6,
+		"BPF_REG_7":           0x7,
+		"BPF_REG_8":           0x8,
+		"BPF_REG_9":           0x9,
+		"BPF_REG_10":          0x10,
+		"MAX_BPF_REG":         0x11,
+		"BPF_NOEXIST":         0x1,
+		"BPF_EXIST":           0x2,
+		"PACKET_MR_MULTICAST": 0,
+		"PACKET_MR_PROMISC":   1,
+		"PACKET_MR_ALLMULTI":  2,
+		"PACKET_MR_UNICAST":   3,
+	}
+)
+
+//GenBuff generates a new buffer that copies bufVal[:size] and then fills zeros
+func GenBuff(bufVal []byte, size uint64) []byte {
```

Comment:
This all can be replaced with:
```
for len(bufVal) < size {
	bufVal = append(bufVal, 0)
}
bufVal = bufVal[:size]
```
and then probably inlined since it's used just once

Reply:
ack